examples/allennlp/classifier.jsonnet

```diff
@@ -28,7 +28,6 @@ local ENCODER = CNN_FIELDS(
   pytorch_seed: 42,
```

Comment:
[question] Why are seeds different from ones in `examples/allennlp/allennlp_simple.py`?

Reply:
Let me just clarify: no specific intention. `getSeed()` in my brain returned different seeds.

Reply:
okay.